### PR TITLE
Doc errors

### PR DESCRIPTION
**Pull request by keckler**
Fixed various grammatical and typographical errors that I found while going through the tutorial docs. 

The only change more complicated than simple typos is the addition of the metadata setting to the SFR tutorial. Without this addition, the input that a user builds by following the tutorial will not run. I added a step in the tutorial to put the needed metadata entry into their settings file and added a yaml anchor to the stored input file so that the code renders in the docs.

### CODE REVIEW

File: doc/tutorials/walkthrough_inputs.rst [hunk 1]
```diff
@@ -353,6 +353,13 @@ Now we need to specify some **settings** that define fundamental reactor
 parameters, as well as modeling approximation options. For this, we make a
 **settings file**, called ``anl-afci-177.yaml``.
 
+First we need to supply some metadata.
+
+.. literalinclude:: ../../armi/tests/tutorials/anl-afci-177.yaml
+    :language: yaml
+    :start-after: begin-metadata
+    :end-before: begin-settings
+
```
Comment (jakehader) [inline doc/tutorials/walkthrough_inputs.rst:362]: If you try to delete this metadata section will the settings load or is this actually required as a part of the settings file?

**Review by jakehader — COMMENTED** [review]
Hi Chris,

Please see my comment and make sure to sign the CLA above.

File: doc/tutorials/walkthrough_inputs.rst (hunk 1 above)
Comment (keckler) [reply doc/tutorials/walkthrough_inputs.rst:362]: For me, ARMI will not run without the metadata entry. 

image

Once I put the metadata entry in there, it runs with no problems. Do you think it should not be required? If so, perhaps we should check on another machine as well.

Comment (jakehader) [reply doc/tutorials/walkthrough_inputs.rst:362]: Okay, thanks! I forgot about this section in the settings being required so this helps.

**Comment by jakehader** [comment]
@keckler - Looks good! Please sign the license/cla agreement and then I can approve and merge in.

**Comment by keckler** [comment]
It is signed and should be ready now.

**Review by jakehader — COMMENTED** [review]
Approved.

**Pull request merged**